### ISSUE TITLE
0.11.0: pytest warning

### ISSUE THREAD

**Issue by kloczek** [prometheus/client_python#670]
twisted 21.2.0 and just normal build, install and test cycle used on building package from non-root account:
- "setup.py build"
- "setup.py install --root </install/prefix>"
- "pytest with PYTHONPATH pointing to setearch and sitelib inside </install/prefix>
```console
+ PYTHONPATH=/home/tkloczko/rpmbuild/BUILDROOT/python-prometheus_client-0.11.0-2.fc35.x86_64/usr/lib64/python3.8/site-packages:/home/tkloczko/rpmbuild/BUILDROOT/python-prometheus_client-0.11.0-2.fc35.x86_64/usr/lib/python3.8/site-packages
+ PYTHONDONTWRITEBYTECODE=1
+ /usr/bin/pytest -ra
=========================================================================== test session starts ============================================================================
platform linux -- Python 3.8.9, pytest-6.2.4, py-1.10.0, pluggy-0.13.1
benchmark: 3.4.1 (defaults: timer=time.perf_counter disable_gc=False min_rounds=5 min_time=0.000005 max_time=1.0 calibration_precision=10 warmup=False warmup_iterations=100000)
rootdir: /home/tkloczko/rpmbuild/BUILD/client_python-0.11.0
plugins: forked-1.3.0, shutil-1.7.0, virtualenv-1.7.0, expect-1.1.0, httpbin-1.0.0, flake8-1.0.7, timeout-1.4.2, betamax-0.8.1, freezegun-0.4.2, case-1.5.3, isort-1.3.0, aspectlib-1.5.2, asyncio-0.15.1, toolbox-0.5, xprocess-0.17.1, aiohttp-0.3.0, checkdocs-2.7.0, mock-3.6.1, rerunfailures-9.1.1, requests-mock-1.9.3, cov-2.12.1, pyfakefs-4.5.0, cases-3.6.1, flaky-3.7.0, hypothesis-6.14.0, benchmark-3.4.1, xdist-2.3.0, Faker-8.8.1
collected 267 items

. .                                                                                                                                                                  [  0%]
tests/test_asgi.py ....                                                                                                                                              [  1%]
tests/test_core.py ..............................................................................                                                                    [ 31%]
tests/test_exposition.py .......................................................                                                                                     [ 51%]
tests/test_gc_collector.py ..                                                                                                                                        [ 52%]
tests/test_graphite_bridge.py .......                                                                                                                                [ 55%]
tests/test_multiprocess.py ......................                                                                                                                    [ 63%]
tests/test_parser.py ........................                                                                                                                        [ 72%]
tests/test_platform_collector.py ..                                                                                                                                  [ 73%]
tests/test_process_collector.py ....                                                                                                                                 [ 74%]
tests/test_twisted.py .                                                                                                                                              [ 74%]
tests/test_wsgi.py .....                                                                                                                                             [ 76%]
tests/openmetrics/test_exposition.py ..................                                                                                                              [ 83%]
tests/openmetrics/test_parser.py ............................................                                                                                        [100%]

============================================================================= warnings summary =============================================================================
tests/test_twisted.py::MetricsResourceTest::test_reports_metrics
  /usr/lib/python3.8/site-packages/twisted/internet/defer.py:662: DeprecationWarning: Using readBody with a transport that does not have an abortConnection method
    current.result = callback(current.result, *args, **kw)

-- Docs: https://docs.pytest.org/en/stable/warnings.html
===================================================================== 267 passed, 1 warning in 11.17s ======================================================================
```

**Comment by karezachen** [comment]
@csmarchbanks PTAL

**Comment by karezachen** [comment]
@csmarchbanks 
Hello Chris，
The pull request I submitted addresses this issue. Could you please assist in merging it?

**Comment by csmarchbanks** [comment]
Fixed in #947
